builder/scripts.d/45-libunibreak.sh

```diff
@@ -0,0 +1,37 @@
+#!/bin/bash
+
+SCRIPT_REPO="https://github.com/adah1972/libunibreak.git"
+SCRIPT_COMMIT="32dc4fabf20176aeaa63c4880cc53cc123e53261"
+
+ffbuild_enabled() {
+    return 0
```

Comment:
libunibreak is already part of the macOS system image dependency: https://github.com/jellyfin/jellyfin-ffmpeg/blob/jellyfin/builder/images/macos/06-libunibreak.sh

If we want to build it here we should also remove it from the system dependency